discovery-provider/src/queries/query_helpers.py

```diff
@@ -314,6 +314,29 @@ def populate_user_metadata(session, user_ids, users, current_user_id, with_track
     return users
 
 
+def get_user_follower_counts(session, user_ids, users):
```

Comment:
just so i understand, these functions have been added so we have the necessary sort fields. and we don't want to add populate_user_metadata since it's too slow

Reply:
yep, exactly. these are just copy pasta from populate_x_data